### PR TITLE
Fix errors reported by user (use osim. where needed,  true ->True)

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #3050 

### CHANGELOG.md (choose one)

- no need to update because bugfix

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3053)

### CODE REVIEW

**Comment by aymanhab** [comment]
Ready for quick review @carmichaelong

File: Bindings/Python/OpenSenseExample/OpenSense_CalibrateModel.py [hunk 1]
```diff
@@ -22,14 +22,14 @@
 
 # Import OpenSim libraries
 import opensim as osim
-
+import math.pi as pi
```
Comment (carmichaelong) [inline Bindings/Python/OpenSenseExample/OpenSense_CalibrateModel.py:25]: This line doesn't work for me (`No module named 'math.pi'; 'math' is not a package`). Consider `from math import pi` instead, and then `pi` will be defined too

File: Bindings/Python/OpenSenseExample/OpenSense_OrientationTracking.py [hunk 2]
```diff
@@ -26,12 +26,12 @@
 
 # Import OpenSim Libraries
 import opensim as osim
-
+import math.pi as pi
```
Comment (carmichaelong) [inline Bindings/Python/OpenSenseExample/OpenSense_OrientationTracking.py:29]: Same here

**Review by carmichaelong — CHANGES_REQUESTED** [review]

**Review by aymanhab — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3053)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @carmichaelong)

___
*[Bindings/Python/OpenSenseExample/OpenSense_CalibrateModel.py, line 25 at r1](https://reviewable.io/reviews/opensim-org/opensim-core/3053#-Mg8GAWLoTjYhV6oXWwQ-r1-25:-MgC0IwN9QAv4XEAhL0h:b-896fix) ([raw file](https://github.com/opensim-org/opensim-core/blob/640e5a1c64ff168ef384f658d70415a88dd8822f/Bindings/Python/OpenSenseExample/OpenSense_CalibrateModel.py#L25)):*
<details><summary><i>Previously, carmichaelong wrote…</i></summary><blockquote>

This line doesn't work for me (`No module named 'math.pi'; 'math' is not a package`). Consider `from math import pi` instead, and then `pi` will be defined too
</blockquote></details>

Done.
___
*[Bindings/Python/OpenSenseExample/OpenSense_OrientationTracking.py, line 29 at r1](https://reviewable.io/reviews/opensim-org/opensim-core/3053#-Mg8GAWLoTjYhV6oXWwR-r1-29:-MgC0KRmEKqI3A0vWw4W:b-896fix) ([raw file](https://github.com/opensim-org/opensim-core/blob/640e5a1c64ff168ef384f658d70415a88dd8822f/Bindings/Python/OpenSenseExample/OpenSense_OrientationTracking.py#L29)):*
<details><summary><i>Previously, carmichaelong wrote…</i></summary><blockquote>

Same here
</blockquote></details>

Done.

**Review by carmichaelong — APPROVED** [review]
LGTM. I will also add this to the testing doc

**Comment by aymanhab** [comment]
Thanks @carmichaelong

**Pull request merged**